.github/workflows/release.yml

```
@@ -35,6 +35,23 @@ jobs:
           checksum_file=$(echo "$ARTIFACTS" | jq -r '.[] | select (.type=="Checksum") | .path')
           echo "::set-output name=hashes::$(cat $checksum_file | base64 -w0)"
 
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: git fetch --prune --unshallow
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+          check-latest: true
+      - uses: imjasonh/setup-ko@v0.6 # This installs the current latest release.
+      - name: push image on ghcr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          tag=$(echo ${{ github.ref }} | cut -c11-) # get tag name without tags/refs/ prefix.
+          ko build --bare --platform=all -t latest -t ${{ github.sha }} -t ${tag} ./
```

```suggestion
      - run: |
          tag=$(echo ${{ github.ref }} | cut -c11-) # get tag name without tags/refs/ prefix.
          ko build --bare --platform=all -t latest -t ${{ github.sha }} -t ${tag} ./
```

The env is unneeded because setup-ko already sets up auth if you're pushing to GHCR, and in general I just drop the `name` if it's pretty clear what the step does.